UH-StudentServices/student_guide
# HUB-624: Update uh_courses_embed

# Update uh_courses_embed module

The embedded courses URL has changed from https://studies.helsinki.fi/ to https://studies.helsinki.fi/syllabus (the old URL has been left intact for now in order to provide time for updates).

Update to get the new URL.

# Testing

- View content by type "Courses embed".
- Verify that a network request is made to the updated URL instead of the old URL and that the embed works.
